rust/abacus-core/src/accumulator/merkle.rs

```diff
@@ -41,7 +41,7 @@ pub enum MerkleTree {
 
 /// A merkle proof object. The leaf, its path to the root, and its index in the
 /// tree.
-#[derive(Debug, Clone, Copy, serde::Deserialize, serde::Serialize, PartialEq)]
+#[derive(Debug, Clone, Copy, serde::Deserialize, serde::Serialize, PartialEq, Eq)]
```

Comment:
curious why the `Eq` here and elsewhere were needed?

Reply:
Because clippy rules changed and now reports a warning when you derive PartialEq but no Eq. Our pipeline errors on _any_ clippy warnings so had to change.